textile/chat-features.textile

```diff
@@ -396,6 +396,7 @@ Broadly speaking, messages are published via REST calls to the Chat HTTP API and
 * @(CHA-M5)@ For a given subscription, messages prior to the point of subscription can be retrieved in a history-like request. Note that this is the point in the message flow @(subscription point)@ at which the subscription was made, NOT the channel attachment point.
 ** @(CHA-M5a)@ @[Testable]@ If a subscription is added when the underlying realtime channel is @ATTACHED@, then the @subscription point@ is the current @channelSerial@ of the realtime channel.
 ** @(CHA-M5b)@ @[Testable]@ If a subscription is added when the underlying realtime channel is in any other state, then its @subscription point@ becomes the @attachSerial@ at the the point of channel attachment.
+*** @(CHA-M5b1)@ @[Testable]@ If the channel fails to attach then an error should be thrown with an error code 102112 (room in invalid state).
```

Comment:
In chat-js, we don't throw an error here - the SDK just waits until we eventually _do_ become attached (we add a `once` listener for attached events).

I think it's worth considering though whether or not we want to specify that calls will fail with an error of `disposed` if the room gets released in the meantime?